cnf-certification-test/accesscontrol/suite.go

```diff
@@ -611,3 +618,32 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+// a test for security context that are allowed from the documentation of the cnf
+// an allowed one will pass the test
+
+func testContainerSCC(env *provider.TestEnvironment) {
+	var badContainer []securitycontextcontainer.PodListcategory
+	var goodContainer []securitycontextcontainer.PodListcategory
+	var listCategoryContainer []securitycontextcontainer.PodListcategory
+	highLevelCat := securitycontextcontainer.CategoryID1
+	for _, pod := range env.Pods {
+		listCategory := securitycontextcontainer.CheckPod(pod)
+		listCategoryContainer = append(listCategoryContainer, listCategory...)
+		for _, cat := range listCategory {
+			if cat.Category > securitycontextcontainer.CategoryID1NoUID0 {
+				badContainer = append(badContainer, cat)
+			} else {
+				goodContainer = append(goodContainer, cat)
+			}
+			if cat.Category > highLevelCat {
+				highLevelCat = cat.Category
+			}
+		}
+	}
+	tnf.ClaimFilePrintf("list of goodContainer", goodContainer)
+	tnf.ClaimFilePrintf("list of badContainers", badContainer)
+	tnf.ClaimFilePrintf("high level of category", highLevelCat)
+	tnf.ClaimFilePrintf("list of all ", listCategoryContainer)
```

Comment:
I think the claim file prints should contain more context.

"Containers that do not fall within SCC Category 1: ... "
"Containers that fall within SCC Category 1: ...." 

Something that is more verbose for the partner to read in the logs.